### PR TITLE
implement vertical split

### PR DESCRIPTION
**Pull request by jesuiswk**

### CODE REVIEW

File: autoload/ctrlsf.vim [hunk 1]
```diff
@@ -227,7 +229,11 @@ func! s:OpenFileInWindow(file, lnum, col, mode, split) abort
 
         if bufname('%') !~# a:file
             if a:split || (&modified && !&hidden)
-                exec 'silent split ' . fnameescape(a:file)
+                if a:split == 2
```
Comment (dyng) [inline autoload/ctrlsf.vim]: Please update comments in `line 214`.

File: autoload/ctrlsf.vim [hunk 2]
```diff
@@ -165,6 +165,8 @@ func! ctrlsf#JumpTo(mode) abort
         call s:OpenFileInWindow(file, lnum, col, 2, 0)
     elseif a:mode ==# 'split'
         call s:OpenFileInWindow(file, lnum, col, 1, 1)
+  elseif a:mode ==# 'vsplit'
```
Comment (dyng) [inline autoload/ctrlsf.vim]: Please indent this line using 4 spaces but not 2 spaces.

**Comment by dyng** [comment]
An awesome pull request. But I think there are extra two things need to be fix:

1. default value of `g:ctrlsf_mapping`. (see [here](https://github.com/dyng/ctrlsf.vim/blob/master/plugin/ctrlsf.vim#L137-L150))
2. documents about `g:ctrlsf_mapping`.

File: plugin/ctrlsf.vim [hunk 3]
```diff
@@ -139,6 +139,7 @@ let s:default_mapping = {
     \ "open"    : ["<CR>", "o"],
     \ "openb"   : "O",
     \ "split"   : "<C-O>",
+    \ "vsplit"  : "<C-o>",
```
Comment (dyng) [inline plugin/ctrlsf.vim]: vim can't distinguish `<C-O>` from `<C-o>`, they share same character code. I think just leave map for vsplit empty should be ok.

Comment (jesuiswk) [inline plugin/ctrlsf.vim]: ok.i map to v for vertical split and h for horizontal split. dont really use visual mode in there...

**Comment by dyng** [comment]
Thanks :) Let's merge.

**Pull request merged**